.env.example

```diff
@@ -0,0 +1 @@
+REACT_APP_HOSPITALRUN_SERVER=http://0.0.0.0:3001
```

Comment:
Change to `HOSPITALRUN_API`. We will use the same war in CD/CI and Kubernetes.

Reply:
Sounds good!!